src/content/en/fundamentals/accessibility/01-focus/focus-three.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: shared/narrow
+title: "Experiencing Focus"
+description: "How different kinds of users experience screen focus"
+published_on: 2016-03-01
+updated_on: 2016-03-01
+order: 3
+translation_priority: 0
+authors:
+  - megginkearney
+  - dgash
+key-takeaways:
+  tldr: 
+    - "Learn what accessibility means and how it applies to web development."
+    - "Learn how to make web sites accessible and usable for everyone."
+    - "Learn how to include basic accessibility with minimal development impace."
+    - "Learn what HTML features are available and how to use them to improve accessibility."
+    - "Learn about advanced accessibility techniques for creating polished accessibility experiences."
+notes:
+  efficiency:
+    - "Understanding the accessibility issue, its scope, and its impact can make you a better web developer."
+  problem-solving:
+    - "Catching, identifying, and removing potential accessibility roadblocks before they happen can improve your development process and reduce maintenance requirements."
+---
+
+Let's try some of the focus techniques we just discussed. Again, using Chrome, go to this <a href="http://robdodson.github.io/udacity-a11y/lesson2-focus/01-basic-form/" target="_blank">airline site mockup page</a> and search for a specific ticket **using only keyboard input**. The page doesn't accept mouse input, so you can't fudge the exercise (not that we don't trust you ;-). 
+
+![Airline site mockup](imgs/airlinesite.png)
```

Comment:
screenshot is outdated (doesn't contain arrival field). grab a new one from http://udacity.github.io/ud891/lesson2-focus/01-basic-form/